examples/rlhf/train_rlhf.py

```diff
@@ -0,0 +1,385 @@
+from copy import deepcopy
+
+import matplotlib.pyplot as plt
+import torch
+import torch.nn.functional as F
+
+from data import get_prompt_dataloader
+from models.actor_critic import init_actor_critic
+from models.reward import init_reward_model
+from tensordict import TensorDict
+from tensordict.nn import TensorDictModuleBase
+from torchrl.data.replay_buffers import (
+    LazyTensorStorage,
+    SamplerWithoutReplacement,
+    TensorDictReplayBuffer,
+)
+from torchrl.objectives import ClipPPOLoss
+from torchrl.objectives.value import GAE
+from tqdm import trange
+from transformers import GenerationConfig, GPT2Tokenizer
+from utils import get_file_logger, load_config
+
+EOS_TOKEN_ID = 50256
+
+
+class VmapCritic(TensorDictModuleBase):
```

Comment:
missing docstring

Reply:
Worth putting in the repo + add tests?

Reply:
Quite possibly, it seems fairly generic.